packages/wdio-types/src/Capabilities.ts

```diff
@@ -155,6 +155,13 @@ export interface VendorExtensions extends EdgeCapabilities, AppiumW3CCapabilitie
         [name: string]: any;
     };
 
+    /**
+     * Selenium 4.0 Specific
+     */
+    'se:options'?: {
+        cdp?: string;
```

Comment:
Do we know more Selenium specific options we can add within this PR?

Reply:
Not to my knowledge